reinitialize-pods/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "linkerd-reinitialize-pods"
+version = "0.1.0"
+authors = ["Linkerd Authors <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+license = "Apache-2.0"
+publish = false
+
+[features]
+default = ["openssl-tls", "openssl-vendored"]
+openssl-tls = ["kube/openssl-tls"]
+# Vendor openssl to statically link lib
+openssl-vendored = ["openssl/vendored"]
+rustls-tls = ["kube/rustls-tls"]
+
+[dependencies]
+anyhow = "1"
+futures-util = "0.3"
+k8s-openapi = { version = "0.19", features = ["v1_21"] }
+kube = { version = "0.85", features = ["runtime"] }
```

Comment:
there are updated versions of k8s-openapi, kube, and kubert available (0.20, 0.87.1, and 0.20)

Reply:
I was hesitant to bump k8s-openapi to 0.20 because its earliest supported k8s is `v1_22` and we still officially support 1.21, but I guess it should be fine, I think we're not using any API here that would break in between those two versions.